kadmin/server.c

```diff
@@ -249,6 +249,30 @@ kadmind_dispatch(void *kadm_handlep, krb5_boolean initial,
 	krb5_store_int32(sp, ret);
 	break;
     }
+    case kadm_prune:{
```

Comment:
We could have implemented prune without a new RPC, but doing it as a new RPC would allow us to have a new privilege for this operation.